modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/parameter/contributor/UserSXPParameterContributor.java

```diff
@@ -534,16 +534,29 @@ private void _contribute(
 			userId);
 
 		if (!userGroups.isEmpty()) {
-			Stream<UserGroup> stream = userGroups.stream();
-
 			sxpParameters.add(
 				new LongArraySXPParameter(
 					"user.user_group_ids", true,
-					stream.map(
-						UserGroup::getUserGroupId
-					).toArray(
-						Long[]::new
-					)));
+					ListUtil.toArray(
```

Comment:
@W2jx You should be able to use `TransformUtil.transformToArray` here, since it asks for a `Long[]`.